### PR TITLE
remove mailchimp

### PR DESCRIPTION
**Pull request by nickpell**
http://manage.dimagi.com/default.asp?183458

### CODE REVIEW

**Comment by nickpell** [comment]
@orangejenny

**Comment by czue** [comment]
:boom: :rage1: :boom:

File: requirements/requirements.txt [hunk 1]
```diff
@@ -72,7 +72,6 @@ twilio==3.6.4
 django-countries==3.3
 reportlab==3.0
 stripe==1.12.2
-mailchimp==2.0.7
```
Comment (czue) [inline requirements/requirements.txt]: should we add this to `uninstall-requirements.txt`? i think otherwise it will still sit in our prod venvs

Comment (nickpell) [inline requirements/requirements.txt]: https://github.com/dimagi/commcare-hq/pull/8513

**Pull request merged**